iatirulesets/text.py

```diff
@@ -33,6 +33,12 @@ def rules_text(rules, reduced_path, show_all=False):
                             out.append('``{0}`` should start with the value in ``{1}``'.format(case_path, case['start']))
                         elif rule == 'regex_matches':
                             out.append('``{0}`` should match the regex ``{1}``'.format(case_path, case['regex']))
+                        elif rule == 'sum':
+                            sum_total = case['sum']
+                            if other_paths:
+                                out.append('The sum of values matched by ``{0}`` and ``{1}`` must be ``{2}``.'.format(case_path, human_list(other_paths, 'and'), sum_total))
+                            else:
+                                out.append('The sum of values matched by ``{0}`` must be ``{2}``.'.format(case_path, sum_total))
```

Comment:
In terms of copy, how about this text for added clarity - either
`The sum of values matched at paths [path] and [path] must be [sum_total]`
`The sum of values contained within [path] and [path] must be [sum_total]`

??